pkg/kube/apis/boshdeploymentcontroller/v1alpha1/types.go

```diff
@@ -8,10 +8,28 @@ import (
 // It's used as input for the Kube code generator
 // Run "make generate" after modifying this file
 
+// Valid values for ref types
+const (
+	ConfigMapType string = "configmap"
+	SecretType    string = "secret"
+	UrlType       string = "url"
+)
+
 // BOSHDeploymentSpec defines the desired state of BOSHDeployment
 type BOSHDeploymentSpec struct {
-	ManifestRef string `json:"manifest-ref"`
-	OpsRef      string `json:"ops-ref"`
+	Manifest Manifest `json:"manifest"`
+	Ops      []Ops    `json:"ops"`
+}
+
+type Manifest struct {
```

Comment:
`make test` complains about missing comments on Manifest. Personally I don't think they make sense, but our CI will fail. We started discussions to change the CI behavior so we can apply linting and vet warnings only when useful.